### PR TITLE
fix some small mistakes in kubeadm

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
fix some small mistakes in kubeadm
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by dixudx** [comment]
Only a small fix.
/lgtm

FYI. We're in code freezing now. Will block on any non-critical changes.

File: cmd/kubeadm/app/cmd/upgrade/apply.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func NewCmdApply(apf *applyPlanFlags) *cobra.Command {
 
 			// If the version is specified in config file, pick up that value.
 			if flags.cfgPath != "" {
-				glog.V(1).Infof("fetching configuration from file", flags.cfgPath)
+				glog.V(1).Infof("fetching configuration from file %s", flags.cfgPath)
```
Comment (luxas) [inline cmd/kubeadm/app/cmd/upgrade/apply.go]: is this applying a newline (\n) automatically?

**Review by luxas — APPROVED** [review]
/lgtm
/approve
with one comment

File: cmd/kubeadm/app/cmd/upgrade/apply.go (hunk 1 above)
Comment (hanxiaoshuai) [reply cmd/kubeadm/app/cmd/upgrade/apply.go]: Yes, it will apply a newline (\n) automatically.

**Pull request merged**